File: HigherLower/backend/HigherLowerBackend/HigherLowerBackend/Card.cs
namespace HigherLowerBackend
{
    public class Card
    {
        public string suit { get; set; }
        public string rank { get; set; }

        public Card(string cardSuit, string cardRank)
        {
            suit = cardSuit;
            rank = cardRank;
        }
    }
}

File: HigherLower/frontend/src/components/Card.js
import React from 'react';

//displays image of a card on screen
export default function Card({suit, rank}) {
    const imageUrl = `/images/fronts/${suit}_${rank}.svg`;
    return (
        <>
          <img className="card" src={imageUrl} alt={`${rank} of ${suit} playing card`}/>
        </>
    );
}